dev/external_database/deploy_mysql.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+default_name="kubecf-mysql"
+printf "Name (%s): " "${default_name}"
+read -r name
+if [ -z "${name}" ]; then
+  name="${default_name}"
+fi
+
+default_namespace="kubecf-mysql"
+printf "Namespace (%s): " "${default_namespace}"
+read -r namespace
+if [ -z "${namespace}" ]; then
+  namespace="${default_namespace}"
+fi
+
+stty -echo
+printf "Root password: "
+read -r root_password
+stty echo
+printf "\\n"
+if [ -z "${root_password}" ]; then
+  >&2 echo "The root password cannot be empty"
+  exit 1
+fi
+
+databases=(
+  "cloud_controller"
+  "diego"
+  "network_connectivity"
+  "network_policy"
+  "routing-api"
+  "uaa"
+  "locket"
+  "credhub"
+)
+
+if ! "{KUBECTL}" get namespace "${namespace}" 1> /dev/null 2> /dev/null; then
+  "{KUBECTL}" create namespace "${namespace}"
+fi
+
+"{HELM}" template "{MYSQL_CHART}" \
+  --name "${name}" \
+  --namespace "${namespace}" \
+  --set "mysqlRootPassword=${root_password}" \
+  --set "testFramework.enabled=false" \
+  | "{KUBECTL}" apply -f - \
+    --namespace "${namespace}"
+
+"{KUBECTL}" wait pod \
+  --for condition=ready \
+  --namespace "${namespace}" \
+  --selector "app=${name}" \
+  --timeout 300s
+
+# Ensure the database is fully functional.
+until echo "SELECT 'Ready!'" | "{KUBECTL}" run mysql-client --rm -i --restart='Never' --image docker.io/mysql --namespace "${namespace}" --command -- \
```

Comment:
You shouldn't need an explicit `docker.io` here. (It's technically `docker.io/library/mysql`, I think?)

Do we need to give an explicit image version (tag) here?

Do we want to import the [`sql-readiness`] stuff here?  And maybe [`database-seeder`]?

[`sql-readiness`]: https://github.com/SUSE/scf/tree/develop/src/scf-release/src/acceptance-tests-brain/sqlreadiness
[`database-seeder`]: https://github.com/SUSE/scf-helper-release/tree/master/src/database-seeder

Reply:
Given that we don't publish the binaries for sql-readiness and database-seeder, and that the current approach on this PR works, I'll punt this to later once we have more time to this enhancement.

Reply:
I removed the `docker.io` registry and pinned the tag.